CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.4.5
+
+- introduced CLEO Modules support
+
```

Comment:
can you remove this line? I'll make a complete changelog for the next version when it's ready